src/content/self-hosted/install/upgrade.mdx

```diff
@@ -61,6 +61,13 @@ registry:
 >
 > If you had a custom value for `cloud.secret`, you need to specify the right values under `registry.secret` key.
 
+Also, we have changed the default behavior of BuildKit to don't use a persistent volume. If your BuildKit instance was already using a PVC, it should be kept but we recommend you to include the following key in the configuration file:
```

Comment:
this guidance is odd. it sounds like we don't know if our feature works or not. if we are not confident the volume will stay, then let's add this to the upgrade guide as a must have.

Reply:
+1 to add it to the upgrade guide. We prefer that our users set that property explicitly moving forward

Reply:
Changed to be the first thing commented in the upgrade to `1.7.x`